mappings/net/minecraft/loot/function/LootFunctionConsumingBuilder.mapping

```diff
@@ -1,4 +1,4 @@
 CLASS net/minecraft/class_116 net/minecraft/loot/function/LootFunctionConsumingBuilder
-	METHOD method_511 withFunction (Lnet/minecraft/class_117$class_118;)Ljava/lang/Object;
-		ARG 1 lootFunctionBuilder
+	METHOD method_511 apply (Lnet/minecraft/class_117$class_118;)Ljava/lang/Object;
```

Comment:
`build`?

Reply:
it's basically applies a loot function to the builder. 

An example of how it's used would be:
```java
private static <T> T applyExplosionDecay(ItemConvertible drop, LootFunctionConsumingBuilder<T> builder) {
   if (!BlockLootTableGenerator.EXPLOSION_IMMUNE.contains(drop.asItem())) {
       return builder.apply(ExplosionDecayLootFunction.builder());
   }
   return builder.getThis();
}
```

Reply:
Ah, thought the return type is the built type, not the self type.